configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
So, you think we should NOT use wxDIB under wxGTK/Win32. From the code ww will have to NOT define wxHAS_IMAGES_IN_RESOURCES or change wxHAS_IMAGES_IN_RESOURCES so wxUSE_WXDIB is NOT needed to be defined to use images in resources.

I just wish to confirm you do NOT think we should use both wxDIB and "images in resources" in wxGTK/Win32?

Tim S.